src/components/Modal/Modal.tsx

```diff
@@ -67,18 +58,21 @@ const ModalContainer = styled('div')<ModalContainerProps>`
       ? 'translateX(-50%) translateY(-50%) scale(1)'
       : 'translateX(-50%) translateY(30%) scale(0.9)'};
 
-  @media (max-height: 900px) {
-    max-height: calc(100vh - 2rem);
+  @media (min-width: 600px) {
```

Comment:
Isn't it quite a narrow gap here between 600px and (what I assume TABLET_BP_UP is) 768px? Is it needed?

Reply:
Good point! In this case `TABLET_BP_UP` is 801px (yep, breakpoints are a bit iffy :D ). But let me check if I can change that breakpoint, 768px makes more sense 👍